.github/ISSUE_TEMPLATE/Platform _Epic_Template.md

```diff
@@ -0,0 +1,24 @@
+---
+Name: Epic template
+About: Template for creating  platform epics
+Title: "[EPIC]"
+Labels: epic platform
+
+---
+
+## Outcome/Objective ##
+
+## Description ##
+
+## Product Requirement(s) ##
+
+## Use Case ##
+
+## Dependencies ##
+
+## Acceptance criteria  ##
+
+## Technical Requirement(s)##
```

Comment:
```suggestion
## Technical Requirement(s)
<!---
- Architecture: What existing patterns and frameworks will be used for this work? What new ones will be added? What additional libraries need to be used?
- Data model: What changes are there to the data model, and how will those changes be implemented (i.e. requires migration)?
- APIs: What existing services are involved? What new ones will be added?
- Integrations: What integrations are involved?
-->
```

Reply:
I've added a comment explaining what are some things that could/should be addressed here. I imagine in some cases I would just link to an SRD, and in others maybe fill out some or all of this, depending on the size and complexity of the epic.